### ISSUE TITLE
Do not fire SetRunningDegarded from Configured

### ISSUE THREAD

**Issue by kakwok** [HCALRunControl/levelOneHCALFM#226]
We see these logs right after getting configured from time to time at P5. The alarmerWatchThread should only fires SetrunningDegraded when FM is in `Running` or `RunningDegraded`. But we did not goToError. 

` 2017-02-03 16:47:56 and 856 ms : cms.hcalpro.rcms.statemachine.StateMachineEngine
     ERROR Transition for input SetRunningDegraded and State Configured not defined
 2017-02-03 16:47:56 and 856 ms : cms.hcalpro.rcms.fm.fw.EventProcessor
     ERROR State Machine error for Input SetRunningDegraded
 2017-02-03 16:47:56 and 857 ms : cms.hcalpro.rcms.fm.fw.FunctionManager
     ERROR Function Manager executeEvent: event processing failure
 2017-02-03 16:47:56 and 857 ms : cms.hcalpro.rcms.fm.fw.FunctionManager
     ERROR execution of event class rcms.statemachine.definition.Input failed`

**Comment by jhakala** [comment]
Could this be solved by only running the alarmerWatchThread during the running/runningDegraded states (i.e. only starting it on Start and then stopping it on Stop)?

**Comment by jhakala** [comment]
We seem to also be getting this similar error message:
```2017-03-20 16:50:04 and 970 ms : cms.hcalpro.rcms.statemachine.StateMachineEngine
ERROR: Transition for input SetRunningDegraded and State Exiting not defined
2017-03-20 16:50:04 and 970 ms : cms.hcalpro.rcms.fm.fw.EventProcessor
ERROR: State Machine error for Input SetRunningDegraded
2017-03-20 16:50:04 and 972 ms : cms.hcalpro.rcms.fm.fw.FunctionManager
ERROR: Function Manager executeEvent: event processing failure
2017-03-20 16:50:04 and 972 ms : cms.hcalpro.rcms.fm.fw.FunctionManager
ERROR: execution of event class rcms.statemachine.definition.Input failed```

**Comment by kakwok** [comment]
Same thing. It just so happened that when the thread wakes up, the state is in Exiting instead of Configured.

**Comment by kakwok** [comment]
Fixed in d61cb4425898f710902d1b0b6e090867948f3575
